src/django/requirements.txt

```diff
@@ -1,3 +1,7 @@
 Django>=3.0,<4.0
 psycopg2-binary>=2.8
 GDAL>=2.2.4,<3.0
+black==19.10b0
```

Comment:
Nice -- we sometimes have a `./scripts/format` to do this, but I think that if you can easily set up auto-format-on-save in your editor, that's probably preferable (though not always easy to integrate with the project structure).